Source/m_menu.c

```diff
@@ -6123,6 +6150,33 @@ void M_ResetSetupMenu(void)
   SetupMenu[0].status = (demo_version < 203) ? 0 : 1;
 }
 
+void M_ResetSetupMenuItems(void)
+{
+  static boolean toggle_boom = false;
```

Comment:
Hm, I don't like the toggle solution and would prefer to set the flags explicitly depending on the currenr `demo_version` alone, not on the value of another variable that was set during the previous call.